docs/firmware/manual-flashing.md

```diff
@@ -6,19 +6,24 @@ Flashing your own firmware has the potential to brick your device. Do not do thi
 
 The steps below assume you are flashing an image named `coreboot.rom`; substitute it as necessary.
 
-1. Download flashrom, decompress, and ensure it is executable:
+1. Download flashrom and cbfstool, decompress, and ensure it is executable:
    * `wget -o flashrom.tar.gz https://mrchromebox.tech/files/util/flashrom_ups_libpci37_20240418.tar.gz`
    * `tar zxf flashrom.tar.gz`
    * `chmod +x flashrom`
+   * `wget https://mrchromebox.tech/files/util/cbfstool.tar.gz && tar -zxf cbfstool.tar.gz && chmod +x cbfstool`
```

Comment:
if you're going to combine 3 commands into one here, should do it above too for consistency